domain/src/mongo.js

```diff
@@ -10,14 +10,18 @@ let db = undefined;
  * @returns {Promise.<Object, Error>} БД
  */
 async function connect(url) {
-    do {
-        try {
-            db = await mongo.connect(url);
-        } catch (e) {
-            continue;
-        }
+    try {
+        db = await mongo.connect(url);
+    } catch (err) {
+        let intervalID = setInterval(async function () {
+            try {
+                db = await mongo.connect(url);
+                clearInterval(intervalID);
```

Comment:
what if an exception happens in the `clearInterval` call? it will be silently ignored by your code, not good
if you `await` the result of this top-level `connect` function, when will it be resolved if mongo connection fails for the first try?

Reply:
let me know if you understand how to fix it, otherwise I'll give more info